adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/BrokerProxy.cs

```diff
@@ -364,17 +364,13 @@ public Intent GetIntentForBrokerActivity(AuthenticationRequest request, Activity
                     intent.PutExtra(BrokerConstants.BrokerRequest, BrokerConstants.BrokerRequest);
                 }
             }
-            catch (OperationCanceledException e)
-            {
-                RequestContext.Logger.Error(e);
-                RequestContext.Logger.ErrorPii(e);
-            }
             catch (Exception e)
             {
                 // Authenticator gets problem from webrequest or file read/write
```

Comment:
>// Authenticator gets problem from webrequest or file read/write [](start = 16, length = 64)

is this comment relevant?

Reply:
maybe...maybe not...doesn't seem relevant now. ⛄️

Reply:
removed.